app/src/main/res/layout/activity_circuit_timer.xml

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <ImageView
+            android:id="@+id/close_button"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:clickable="true"
+            android:focusable="true"
+            android:padding="@dimen/medium_element_padding"
+            android:src="@drawable/ic_close_black" />
+
+        <TextView
+            android:id="@+id/name"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_centerHorizontal="true"
+            android:layout_marginHorizontal="@dimen/medium_element_padding"
+            android:layout_marginVertical="@dimen/xl_large_element_padding"
+            android:textSize="@dimen/huge_text" />
+
+        <TextView
+            android:id="@+id/state"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/name"
+            android:layout_centerHorizontal="true"
+            android:layout_margin="@dimen/medium_element_padding"
+            android:textSize="@dimen/large_text" />
+
+        <TextView
+            android:id="@+id/set"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_below="@+id/state"
+            android:layout_centerHorizontal="true"
+            android:layout_margin="@dimen/medium_element_padding"
+            android:textSize="@dimen/large_text" />
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_alignParentBottom="true"
+            android:orientation="vertical">
+
+            <RelativeLayout
+                android:id="@+id/timerlayout"
```

Comment:
timer_layout following rest of the file convention

Reply:
Yeah gonna have to do this everywhere tbh because I believe back in Feb I did camelCase for layouts and now I stuck with underscores lol, gonna make a story to change everything to underscores.